utils/cache_manager.py

```
@@ -1,20 +1,131 @@
-# Import the Cache class from the flask_caching module
-from flask_caching import Cache
-# Import the cache_config dictionary from the utils.cache_config module
-from utils.cache_config import cache_config
+import json
+import functools
+import hashlib
+from bson import ObjectId
```

Ruff (0.8.0)

`bson.ObjectId` imported but unused

Remove unused import: `bson.ObjectId`

(F401)